go.mod

```diff
@@ -3,9 +3,7 @@ module github.com/newrelic/nri-f5
 go 1.14
 
 require (
-	github.com/davecgh/go-spew v1.1.2-0.20180830191138-d8f796af33cc // indirect
-	github.com/golangci/golangci-lint v1.39.0
-	github.com/newrelic/infra-integrations-sdk v3.7.2+incompatible
-	github.com/pmezard/go-difflib v1.0.1-0.20181226105442-5d4384ee4fb2 // indirect
-	github.com/stretchr/testify v1.7.0
+	github.com/golangci/golangci-lint v1.46.2
```

Comment:
could you move the dependency to the tools folder? https://github.com/newrelic/nri-nginx/pull/89